uwblueprint/marillac-place
# Creation of List-View UI for Schedule Page

## Notion ticket link
[Schedule Page - List-View UI](https://www.notion.so/uwblueprintexecs/Schedule-Page-List-View-UI-aa2b1286476a4fe8a99cd0c69aa19ce2)


## Implementation description
* Utilized mock data to create a list schedule UI for both weekly task and daily tasks, similar to the tasks list.
* Added ScheduleTable to provide a common table for the two schedules (weekly and daily) taking in the mock data and generating a table.
* Utilized the ScheduleTable and created a tabs list for the daily tasks in order to be able to switch between days. Currently we have a map of days in the enum to their data mock, this would need to be changed to get the actual date's data, potentially in a different way.


## Steps to test
1. Go to schedule page, scroll down past the cards, and you should see two tables. For example, this is the table for Daily Tasks:
![image](https://github.com/user-attachments/assets/8b8807b7-6d47-4f78-92c8-36b141277611)
2. If you click between tabs the data should change for some days (although some of the days are the same).


## What should reviewers focus on?
* Make sure all indenting styling is okay on their end ("Weekly Tasks" and "Daily Tasks" left aligned with table, tabs on the Daily Tasks are aligned on top of the table, etc).

## Checklist
